docs/sources/reference/run.md

```diff
@@ -873,18 +873,30 @@ this driver.
 Default logging driver for Docker. Writes JSON messages to file. `docker logs`
 command is available only for this logging driver
 
+The following logging options are supported for this logging driver: [none]
+
 #### Logging driver: syslog
 
 Syslog logging driver for Docker. Writes log messages to syslog. `docker logs`
 command is not available for this logging driver
 
+The following logging options are supported for this logging driver:
+
+    --log-opt address=[tcp|udp|unix]://[host:port|path]
```

Comment:
i would split this in two for tcp|udp and unix, because of the host:port|path part